.github/workflows/ci.yml

```diff
@@ -0,0 +1,41 @@
+name: Go CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    # Setup Go environment
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: '^1.21.x'
+
+    # Checkout the current repository
+    - name: Check out current repository
+      uses: actions/checkout@v4
+
+    # Clone the 'cone' repository
+    - name: Clone 'cone' repository
+      run: git clone https://github.com/ConductorOne/cone.git
+
+    # Clone the 'conductorone-sdk-go' repository
+    - name: Clone 'conductorone-sdk-go' repository
+      run: git clone https://github.com/conductorone/conductorone-sdk-go.git
+
+    # Replace the vendored SDK in 'cone' with the checked-out version
+    - name: Replace vendored SDK
+      run: |
+        rm -rf cone/vendor/github.com/conductorone/conductorone-sdk-go
+        cp -r conductorone-sdk-go cone/vendor/github.com/conductorone/conductorone-sdk-go
```

Comment:
You might want to use `cp -a` to preserve permissions. I don't know if certain things need to be chmodded executable to work.